cachelib/allocator/MarginalHitsState-inl.h

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#include <cstdint>
+
```

Comment:
`-inl.h` is considered as part of `.h` and we do not have any `include` directives in `-inl.h`. Could you move this to `MarginalHitsState.h`?

Reply:
Sure, that’s fine. I’ll amend this and force-push.